layers/+lang/c-c++/packages.el

```diff
@@ -90,7 +89,7 @@
 
 (defun c-c++/init-clang-format ()
   (use-package clang-format
-    :init (spacemacs//c-c++-setup-clang-format)))
+    :hook ((c-mode-local-vars c++-mode-local-vars) . spacemacs//c-c++-setup-format)))
```

Comment:
This is probably problematic, in this layer the keybindings for formating has been added to `spacemacs//c-c++-setup-format` in this case it would set the bindings each time we activate the mode. Please move the keys into the init block of `c-c++/init-clang-format`.

Also be aware that each call to this function adds more hooks to `c-c++-mode-hooks`.